tests/StrictQuoteLineTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace ClickHouseDB\Tests;
+
+use PHPUnit\Framework\TestCase;
+use ClickHouseDB\Quote\StrictQuoteLine;
+
+/**
+ * Class StrictQuoteLineTest
+ * @package ClickHouseDB\Tests
+ */
+class StrictQuoteLineTest extends TestCase
+{
+    use WithClient;
+
+    public function setUp()
+    {
+        $this->client->write('DROP TABLE IF EXISTS cities');
+        return $this->client->write('
+            CREATE TABLE IF NOT EXISTS cities (
+                date Date,
+                city String,
+                keywords Array(String),
+                nums Array(UInt8)
+            ) ENGINE = MergeTree(date, (date), 8192)
+        ');
+    }
+
+    /**
+     * @group test
+     */
+    public function testQuoteValueCSV()
+    {
+        $strict = new StrictQuoteLine('CSV');
+
+        $rows = [
+            ['2018-04-01', '"That work"', ['\"That does not\"', 'That work'], [8, 7]],
```

Comment:
I'm being a bit pedant maybe but `That works` it correct in english

Reply:
And maybe I'd suggest to replace "work"/"does not work" with description what is the purpose of string in the dataset. Because after merging this, it would all be working.

I for example propose `['\"double quoted string\"', 'single qouted string']`. Or just simply put there random strings.